data/quarg/quarg news.txt

```diff
@@ -0,0 +1,425 @@
+news "anonymous quarg"
+	location
+		government "Quarg"
+		attributes "human quarg" "hai quarg"
+	name
+		word
+			"Anonymous Quarg"
+	message
+		word
+			`"Are you what is called a 'tourist'?"`
+	message
+		word
+			`"`
+		word
+			"Are you"
+		word
+			" "
+		word
+			"a merchant"
+			"a merchant captain"
+			"a travelling merchant"
+			"a captain"
+		word
+			"?"
+		word
+			" "
+		word
+			"People"
+		word
+			" "
+		word
+			"of peace"
+			"of law"
+			"of trade"
+		word
+			" "
+		word
+			"are"
+		word
+			" "
+		word
+			"welcome"
+			"appreciated"
+		word
+			" "
+		word
+			"here"
+			"on Quarg worlds"
+			"on all Quarg worlds"
+		word
+			"."
+		word
+			`"`
+news "linguist on quarg planets in human and hai space"
+	location
+		government "Quarg"
+		attributes "human quarg" "hai quarg"
+	name
+		word
+			"Linguist"
+	message
+		word
+			`"`
+		word
+			"The Quarg language is so different from ours"
+		word
+			" "
+		word
+			"that"
+		word
+			" "
+		word
+			"most people"
+			"ordinary people"
+		word
+			" "
+		word
+			"could never"
+			"wouldn't be able to"
+			"would never"
+		word
+			" "
+		word
+			"learn it"
+			"understand it"
+			"speak it"
+		word
+			`"`
+	message
+		word
+			`"`
+		word
+			"I've been studying"
+			"I've studied"
+			"I've been learning"
+		word
+			" "
+		word
+			"the Quarg language"
+		word
+			" "
+		word
+			"for a decade"
+			"for years"
+			"for the most part of my career"
```

Comment:
```suggestion
			"for most of my career"
```

Reply:
I don't actually see what's wrong with that

Reply:
You can leave it like that if you want.